eng/pipelines/templates/steps/analyze.yml

```diff
@@ -3,16 +3,11 @@ parameters:
   ServiceDirectory: ''
   TestMarkArgument: ''
   AdditionalTestArgs: ''
+  Artifacts: []
 
 steps:
   - template: /eng/pipelines/templates/steps/analyze_dependency.yml
 
-  - task: PythonScript@0
```

Comment:
@praveenkuttappan Should we remove this py change log script now?

Reply:
Yes. We can remove it once analyze uses new change log check. Actually that's one of the issue in my list to remove duplicate change log check,.